client/sites/hosting/features.ts

```diff
@@ -7,10 +7,11 @@ import { getSelectedSite } from 'calypso/state/ui/selectors';
 import { AppState } from 'calypso/types';
 
 export function areHostingFeaturesSupported( site?: SiteExcerptData | null ) {
-	const isAtomicSite = !! site?.is_wpcom_atomic || !! site?.is_wpcom_staging_site;
+	const isAtomicOrStagingSite = !! site?.is_wpcom_atomic || !! site?.is_wpcom_staging_site;
```

Comment:
You could perhaps even change the name to this

```suggestion
	const isWoAOrStagingSite = !! site?.is_wpcom_atomic || !! site?.is_wpcom_staging_site;
```

Since flex sites are atomic, but they're just not WoA.